Add working unit tests for `--clients`

portante opened the pull request.

ldoktor committed:
Add support for --clients to pbench-linpack

similarly to other bench scripts let's allow execution on
various/multiple clients.

Signed-off-by: Lukáš Doktor <ldoktor@redhat.com>

MVarshini committed:
Removed package-lock.json

portante committed:
Add working unit tests for `--clients`

webbnh commented on agent/bench-scripts/tests/pbench-linpack/test-66.post:
```diff
@@ -0,0 +1 @@
+test-64.post
```
Since `agent/bench-scripts/tests/pbench-linpack/test-66.pre` references `test-65.pre`, shouldn't this reference `test-65.post`?  (It would be nice to keep things from getting _too_ tangled....)

webbnh replied:
[This has been resolved; I don't see any way to mark it as such on this page, though.]

webbnh reviewed (approved):
One small suggestion-question.

portante reviewed (commented):
Addressed @webbnh's comments.

webbnh reviewed (approved):
👍

portante committed:
Record the Tool Meister data in Redis

In order to enable other work that relies on the Tool Meister data
collected by the Tool Data Sink, we set the "tool-meister-data" key
in Redis without any other changes.

This pulls out a very small portion of work from PR #2659.

riya-17 committed:
Terminating benchmark run if tool rpm does not exist

The `pbench-tool-meister-start` reviews the "tool-meister-data" stored
in the Redis server to identify any tool that failed its installation
check, reporting that to the user.

Fixes #2347.

Co-authored-by: Peter Portante <peter.portante@redhat.com>

portante commented:
Moved to https://github.com/distributed-system-analysis/pbench/pull/2746.
